src/components/Tooltip/Tooltip.tsx

```diff
@@ -136,6 +138,10 @@ interface TooltipProps {
      * Force the tooltip to always be visible, regardless of user interaction
      */
     alwaysVisible?: boolean;
+    /**
+     * The node(s) passed as children will be used as the anchor point for the tooltip
+     */
+    children: ReactNode;
 }
 
 const Tooltip: React.FC<TooltipProps> = ({
```

Comment:
I think it is better to rely on React types and add children here instead of directly to the `TooltipProps`:
```suggestion
const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
```